skimage/io/tests/test_pil.py

```diff
@@ -45,18 +45,26 @@ def test_png_round_trip():
     fname = f.name
     f.close()
     I = np.eye(3)
-    imsave(fname, I)
+    with expected_warnings(['Possible precision loss']):
+        imsave(fname, I)
     Ip = img_as_float(imread(fname))
     os.remove(fname)
     assert np.sum(np.abs(Ip-I)) < 1e-3
 
 
+def test_img_as_grey_flatten():
+    img = imread(os.path.join(data_dir, 'color.png'), as_gray=True)
+    with expected_warnings(['deprecated']):
+        img_flat = imread(os.path.join(data_dir, 'color.png'), flatten=True)
+    assert_array_equal(img, img_flat)
```

Comment:
very nice! Although rename `as_grey_flatten` to `as_gray_flatten`. ;)

Reply:
done